node_modules/oae-core/upload/js/upload.js

```diff
@@ -471,7 +484,7 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
                 }
 
                 // Disable editing on upload
-                $('#upload-modal *', $rootel).prop('disabled', true);
+                $toDisable().prop('disabled', true);
```

Comment:
As this is the only place where we do need the file input element exclude, I'm going to just move it back out of its function and put it here.